docs/DeveloperGuide.md

```diff
@@ -2,37 +2,254 @@
 
 ## Acknowledgements
 
-{list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well}
-
 ## Design & implementation
 
-{Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
+[comment]: <> (//@@author Jeraldchen)
+
+### User Interface
+![UIClass.png](diagrams%2FUIClass.png)
```

Comment:
Should the class diagram include the connection to different classes and use the correct syntax? Perhaps it should include the private and public accessibility in the class diagram